### PR TITLE
data: JohnnyLegault

### PR DESCRIPTION
**Pull request by JohnnyLegault**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5188"></a>

### CODE REVIEW

File: data/JohnnyLegault.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "name": "Jonathan Legault LMT",
+  "displayStatsPublic":true,
+  "type": "personal",
+  "bio": "Hello, I am a Massage Therapist Ready to take on New Clients.",
+  "avatar": "https://github.com/JohnnyLegault.png",
```
Comment (SurajPratap10) [inline data/JohnnyLegault.json:6]:
```suggestion
  "type": "personal",
  "bio": "Hello, I am a Massage Therapist Ready to take on New Clients.",
 
```

Comment (SurajPratap10) [reply data/JohnnyLegault.json:6]: We don't need the `display` and `avatar` after the new update so remove both of these lines.

Comment (JohnnyLegault) [reply data/JohnnyLegault.json]: Thanks, I just fixed them.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**